packages/ui-components/Common/LanguageDropDown/index.tsx

```diff
@@ -23,7 +23,11 @@ const LanguageDropdown: FC<LanguageDropDownProps> = ({
   return (
     <DropdownMenu.Root>
       <DropdownMenu.Trigger asChild>
-        <button className={styles.languageDropdown} aria-label={ariaLabel}>
+        <button
+          className={styles.languageDropdown}
+          aria-label={ariaLabel}
+          data-testid="language-selector"
```

Comment:
why wouldn't we use a locator here based on the label? it's available, isn't it? I suppose it is slightly brittle, because it's the test associating to a string (but so is this)